modules/workers/locals.tf

```diff
@@ -104,7 +104,7 @@ locals {
             "${var.tag_namespace}.pool"               = pool_name,
             "${var.tag_namespace}.cluster_autoscaler" = pool.allow_autoscaler ? "allowed" : "disabled",
           },
-          pool.autoscale ? { "${var.tag_namespace}.oraclecloud.com/cluster_autoscaler" = "managed" } : {},
+          pool.autoscale ? { "${var.tag_namespace}/cluster_autoscaler" = "managed" } : {},
```

Comment:
you introduced a discrepancy in the convention; If you look at the expected behaviour of the screenshot [here](https://github.com/oracle-terraform-modules/terraform-oci-oke/issues/808#issue-1868012025), the normal delimiter is a "dot", not "slash" !

should have been : `${var.tag_namespace}.cluster_autoscaler`

Reply:
and also not consistent with your other PR https://github.com/oracle-terraform-modules/terraform-oci-oke/pull/812